science/digital_rf/Portfile

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+PortGroup           cxx11 1.1
+
+github.setup        MITHaystack digital_rf 2.6.0
+maintainers         {mit.edu:rvolz @ryanvolz} {mit.edu:swoboj @jswoboda} openmaintainer
+description         Read, write, and interact with data in the Digital RF and Digital Metadata formats.
+long_description    The Digital RF project encompasses a standardized HDF5 format for reading and writing of radio frequency data and the software for doing so. The format is designed to be self-documenting for data archive and to allow rapid random access for data processing. For details on the format, refer to the 'documents' directory in the source tree.
+categories          science
+license             BSD
+platforms           darwin
+dist_subdir         digital_rf
+
+checksums           rmd160  f095ff1a80f7175f57a45048c826fe68b6dd56e9 \
+                    sha256  6468418a06c269964c9a5c5f145f7bfebf951226e3a85977226c7967fa50e691 \
+                    size    4528350
+
+configure.ldflags-delete -L${prefix}/lib
+
+# dependencies
+depends_lib-append port:py27-numpy \
+                   port:py27-matplotlib \
+                   port:py27-scipy \
+                   port:py27-pandas \
+                   port:py27-pkgconfig \
+                   port:hdf5 \
+                   port:py27-packaging \
+                   port:py27-dateutil \
+                   port:py27-h5py \
+                   port:py27-six \
+                   port:py27-tz \
+                   port:py27-watchdog \
+                   port:py27-mako \
+                   port:uhd \
+                   port:boost \
+                   port:gnutls \
+                   port:gnuradio
```

Comment:
Please sort the dependencies.

Reply:
I take it you want to sort by requirements of the ports?

Reply:
I would think alphabetically?

Reply:
Yes, alphabetically.